osu.Game/Graphics/OsuFont.cs

```diff
@@ -20,6 +20,41 @@ public static class OsuFont
         /// </summary>
         public static FontUsage Default => GetFont();
 
+        /// <summary>
+        /// Equivalent to Torus with 32px size and semi-bold weight.
+        /// </summary>
+        public static FontUsage Display => GetFont(size: 32, weight: FontWeight.SemiBold);
+
+        /// <summary>
+        /// Equivalent to Torus with 28px size and semi-bold weight.
+        /// </summary>
+        public static FontUsage Subdisplay => GetFont(size: 28, weight: FontWeight.SemiBold);
+
+        /// <summary>
+        /// Equivalent to Torus with 22px size and semi-bold weight.
+        /// </summary>
+        public static FontUsage Heading => GetFont(size: 22, weight: FontWeight.SemiBold);
+
+        /// <summary>
+        /// Equivalent to Torus with 18px size and semi-bold weight.
+        /// </summary>
+        public static FontUsage Subheading => GetFont(size: 18, weight: FontWeight.SemiBold);
+
+        /// <summary>
+        /// Equivalent to Torus with 16px size and regular weight.
+        /// </summary>
+        public static FontUsage Body => GetFont(size: 16, weight: FontWeight.Regular);
+
+        /// <summary>
+        /// Equivalent to Torus with 14px size and regular weight.
+        /// </summary>
+        public static FontUsage Caption => GetFont(size: 14, weight: FontWeight.Regular);
+
+        /// <summary>
+        /// Equivalent to Torus with 12px size and regular weight.
+        /// </summary>
+        public static FontUsage Tiny => GetFont(size: 12, weight: FontWeight.Regular);
```

Comment:
Having these next to definitions like "Numeric" doesn't work well for me. Maybe all these should be prefixed with `Default` or `DefaultFor` or something else.

Alternatively we remove or move the `Numeric` and `Torus` definitions below. Or move the new definitions somewhere else.

Open to suggestions.

Reply:
I'm in favour of removing all the properties below it as I foresee all components in the game using only any of the specifications defined here.

However, removing will be a tough process to go through, so I would at least temporarily prefix those with `Legacy`/`Old` so it's clear that they shouldn't be used anymore.

For existing usages of these definitions, I foresee them using the new specifications with the typeface swapped with alternate, i.e. `OsuFont.Title.With(typeface: FontWeight.TorusAlternate)` or `OsuFont.Body.With(typeface: FontWeight.Numeric)`.